core/src/main/java/hudson/util/ProcessTree.java

```diff
@@ -2242,6 +2242,7 @@ Object writeReplace() {
      * in case there's a fatal problem.
      *
      */
+    @SuppressFBWarnings(value = "MS_PKGPROTECT", justification = "for script console")
```

Comment:
Actually Groovy will just ignore access modifiers and let you mutate a `private` field too.

Reply:
Good point. I checked `usage-in-plugins` and these are unused, so I marked them as package-private and removed the suppression. I also verified that they're still settable from the script console.